setup.py

```diff
@@ -0,0 +1,51 @@
+import os
+import subprocess
+import multiprocessing
+from distutils.core import setup
+from distutils.errors import LibError
+from distutils.command.build import build as _build
+
+class build(_build):
+    @staticmethod
+    def _configure():
+        if subprocess.call(['./configure']) != 0:
```

Comment:
My understanding is that `setup.py` is always run with its directory as the current working directory. Most of the setup.pies I've seen (not that many, tbh) make this assumption. That being said, it's not too hard to fix.

Good point about the Python executable!

I made `setup.py` path-aware and custom Python-aware (in terms of `mk_util.py`) in commit a7ab040cdf1865585bdad6f2d294d744db94b63d.